src/converters/lintConfigs/rules/ruleConverters/eslint-plugin-rxjs/no-async-subscribe.ts

```diff
@@ -4,7 +4,7 @@ export const convertNoAsyncSubscribe: RuleConverter = () => {
     return {
         rules: [
             {
-                ruleName: "no-async-subscribe",
+                ruleName: "rxjs/no-async-subscribe",
```

Comment:
Nice catch, I didn't saw that while reading the [ eslint-plugin-rxjs readme](https://github.com/cartant/eslint-plugin-rxjs) as I jumped right into the rule definition.

Reply:
Same for me, I think that the doc is a bit misleading in that regard, especially [Rules](https://github.com/cartant/eslint-plugin-rxjs#rules) as there is no "rxjs/" prefix there